test/e2e/pages/vl-proza-message.page.js

```diff
@@ -14,6 +15,39 @@ class VlProzaMessagePage extends Page {
         return this._getProzaMessage('#message-2');
     }
 
+    async getToaster() {
+        const toaster = await this.driver.executeScript(
+            `return document.querySelector('[is="vl-toaster"]')`);
+        if (toaster) {
+            return await new VlToaster(this.driver, toaster);
```

Comment:
await is denk ik niet nodig:

```suggestion
            return new VlToaster(this.driver, toaster);
```